src/profile-logic/comparison.js

```diff
@@ -44,6 +44,9 @@ export function mergeProfiles(
   resultProfile.meta.interval = Math.min(
     ...profiles.map(profile => profile.meta.interval)
   );
+  resultProfile.meta.symbolicated = profiles.every(
+    profile => profile.meta.symbolicated
```

Comment:
good catch, I wrote this before I implemented an "unknown" type.

Reply:
```js
  // If all profiles have an unknown symbolication status, we keep this unknown
  // status for the combined profile. Otherwise, we mark the combined profile
  // symbolicated only if all profiles are, so that a symbolication process will
  // be kicked off if necessary.
  if (profiles.some(profile => profile.meta.symbolicated !== undefined)) {
    resultProfile.meta.symbolicated = profiles.every(
      profile => profile.meta.symbolicated
    );
  }
```